src/Command/FindAndUploadFilesCommand.php

```diff
@@ -233,15 +278,32 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $progressBar->finish();
         $io->newLine(2);
 
+        $successfullyCreatedZip = $zip->close();
+        if ($successfullyCreatedZip === false && $uploadAllFiles === true) {
+            $io->warning('Failed to create zip file');
+        } elseif ($uploadAllFiles === true) {
+            $io->success('Successfully created zip file');
```

Comment:
Change to `info` instead. I feel like we should use `success` only on the final output :)